src/lib/OpenEXRUtil/ImfImageChannel.h

```diff
@@ -18,9 +18,9 @@
 #include "ImfUtilExport.h"
 
 #include <IexBaseExc.h>
```

Comment:
what about this one?

Reply:
Yes, missed that one, and another reference to <IexBaseExc> in ImfAttribute.h. I pushed another commit to fix these.